client/package.json

```diff
@@ -21,6 +26,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "toolbox": "react-toolbox-themr"
```

Comment:
remove anything eject and react-toolbox related
afterwards be sure to run 'yarn' on the command line to update the yarn.lock file